src/core/mount.c

```diff
@@ -50,6 +50,7 @@ static const UnitActiveState state_translation_table[_MOUNT_STATE_MAX] = {
 
 static int mount_dispatch_timer(sd_event_source *source, usec_t usec, void *userdata);
 static int mount_dispatch_io(sd_event_source *source, int fd, uint32_t revents, void *userdata);
+static int mount_process_proc_self_mount_info(Manager *m);
```

Comment:
Just a nit, in case we want to be consistent with kernel naming this should be called,

`mount_process_proc_self_mountinfo`

Reply:
very good point. will fix